### PR TITLE
Install postgresql libraries on kinto init (fixes #313)

### LINKED ISSUE #313: kinto init should install postgresql dependencies

### PR DESCRIPTION
**Pull request by leplatrem**
@Natim what do you think? 

It might not be the ideal solution, but it would still help a lot newcomers...

### CODE REVIEW

**Comment by Natim** [comment]
We might install the wrong cliquet version.

**Comment by leplatrem** [comment]
But that command is what we suggest to do in the docs :/

**Comment by Natim** [comment]
Yes let's do that in anycase it will just try that if dependencies are not installed yet so we should be fine.

File: kinto/__main__.py [hunk 1]
```diff
@@ -65,6 +67,16 @@ def main(args=None):
 
         init(config_file, backend)
 
+        # Install postgresql libraries if necessary
+        if backend == "postgresql":
+            try:
+                import psycopg2  # NOQA
+            except ImportError:
+                ret = call('pip install cliquet[postgresql]'.split())
```
Comment (Natim) [inline kinto/__main__.py]: Note that if the venv is not activated it will fail. Cannot we call pip using a python command?

Comment (Natim) [inline kinto/__main__.py]: http://stackoverflow.com/a/15950647/186202

**Comment by Natim** [comment]
Oh yeah.

**Comment by leplatrem** [comment]
\o/

**Pull request merged**